pinot-common/src/main/proto/mailbox.proto

```diff
@@ -22,6 +22,12 @@ syntax = "proto3";
 package org.apache.pinot.common.proto;
 
 service PinotMailbox {
+  // Ping rpc, used to keep the connection active
+  // In future we may also use this to negotiate the mailbox protocol version
```

Comment:
We don't have the concept of mailbox protocol versions today right (so is this talking about a hypothetical scenario in the future where we have better backward compatibility mechanisms)?

Reply:
Yes, it is a hypothetcal scenario